src/Giraffe/FormatExpressions.fs

```diff
@@ -83,20 +83,13 @@ let private convertToRegexPatternAndFormatChars (mode : MatchMode) (formatString
     |> convert
     |> (fun (pattern, formatChars) -> formatRegex mode pattern, formatChars)
 
-/// **Description**
-///
+/// <summary>
 /// Tries to parse an input string based on a given format string and return a tuple of all parsed arguments.
-///
-/// **Parameters**
-///
-/// `format`: The format string which shall be used for parsing.
-/// `input`: The input string from which the parsed arguments shall be extracted.
-/// `options`: The options record with specifications on how the matching should behave.
-///
-/// **Output**
-///
-/// Matched value as an option of 'T
-///
+/// </summary>
+/// <param name="format">The format string which shall be used for parsing.</param>
+/// <param name="options">The options record with specifications on how the matching should behave.</param>
+/// <param name="input">The input string from which the parsed arguments shall be extracted.</param>
+/// <returns>Matched value as an option of 'T</returns>
 let tryMatchInput (format : PrintfFormat<_,_,_,_, 'T>) (options : MatchOptions) (input : string) =
```

Comment:
Screenshot shows that it shows the generic args as a bit odd but again might be my noob understanding. It does this in VSCode and Rider. <img width="658" alt="Screenshot 2020-06-07 at 19 56 52" src="https://user-images.githubusercontent.com/105126/83977548-39e73c00-a8f9-11ea-9c14-8d1d20a1ae42.png">

Reply:
This is what it looks like for me in Ionide/VSCode:

![image](https://user-images.githubusercontent.com/573979/84056542-faadff00-a97b-11ea-9520-e871108887c3.png)

That seems reasonable, I'm not quite sure what you're seeing in terms of generic args wierdness? Do you mean the `'a, 'b, 'c, 'd` parameters that aren't explicitly specified in the code but are implicitly provided by the compiler?

Reply:
I mean after my changes in Rider and I assume rider it has `e is a, f is b`